receiver/mongodbreceiver/testdata/certs/ca-1.cert

```diff
@@ -0,0 +1,20 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
We have tooling to autogenerate certs. Is it possible to hook this into the [Makefile](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/0a6d3531e4d4e992b4936f7a9a4b55092a41efc4/Makefile#L356)?

Reply:
Ahh good catch, also noticed a typo in the Makefile or directory structure for mongodbatlas: 

`receiver/mongodbatlasreceiver/testdata/alerts/certs` should be => `receiver/mongodbatlasreceiver/testdata/alerts/cert`

I can file a separate issue or I can fix it in this PR but just noticed, do you have a preference on which approach?

Reply:
Nice catch. Do you mind creating a separate PR?

Reply:
Yep accidentally committed to this branch, I'll fix that change